crates/proof-of-sql/src/sql/parse/error.rs

```diff
@@ -139,6 +139,13 @@ pub enum ConversionError {
         /// The underlying error
         error: String,
     },
+
+    #[snafu(display("Unsupported unary operator: {op}"))]
+    /// Unsupported unary operation
+    UnsupportedUnaryOperator {
```

Comment:
I think a general `Unsupported` error would be good. Let's just put what we don't support from sqlparser in the error message.

Reply:
Done